packages/web/nginx.conf

```diff
@@ -1,5 +1,9 @@
 server {
   listen 80;
+  server_name dev.thehub-aubg.com ;
+  return 301 $host$request_uri;
```

Comment:
Missing https! This will be a circular redirect. 

Please, refer to this - as closely as possible -> [Done right reference](https://github.com/AUBGTheHUB/hub-conf-signup/blob/master/packages/web/nginx.conf)

Make sure you test your changes

Reply:
Made requested changes